### PR TITLE
fixed text buttons styling

### LINKED ISSUE #370: Text buttons don't follow the Figma designs
## Describe the bug
1. The label "report this content" uses a different text style
2. During hover state, it moves slightly to the right.
3. During hover state, it should show an underline in the label.
4. It doesn't have the 8px spacing parameter to separate it from the above paragraph.

Screen Shot 2020-05-14 at 11 47 05

## Expected behavior
Use the correct hover style and interaction, the text style, and adding a spacing parameter. See the mockup in [Figma (mockup S3)](https://www.figma.com/file/KEXxItd8EloJGbXswuUDqq/Mockup---S3-(License-Language-Changes)?node-id=205%3A60&viewport=6496%2C3630%2C1).

### PR DESCRIPTION
**Pull request by Dishebh**
## Fixes

Fixes #370  by @panchovm 

## Description

Fixed styling of text buttons as per issue description

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by Dishebh** [comment]
@dhruvkb @panchovm @hugosolar  Pls review.

File: src/styles/button.scss [hunk 1]
```diff
@@ -68,10 +68,14 @@ $button-family: $family-alternative;
   }
 
   &.is-text {
+    font-family: Source Sans Pro;
+    font-style: normal;
+    line-height: 19px;
     text-decoration: none;
     &:hover, &:active, &:focus {
-      border: none;
+      border-color: transparent;
```
Comment (dhruvkb) [inline src/styles/button.scss]: Changing `border-color` to transparent seems like a hack rather than a proper fix.

File: src/styles/button.scss [hunk 2]
```diff
@@ -68,10 +68,14 @@ $button-family: $family-alternative;
   }
 
   &.is-text {
+    font-family: Source Sans Pro;
```
Comment (dhruvkb) [inline src/styles/button.scss]: Use a token from `typography.scss`.

File: src/styles/button.scss [hunk 3]
```diff
@@ -68,10 +68,14 @@ $button-family: $family-alternative;
...
+    font-style: normal;
+    line-height: 19px;
```
Comment (dhruvkb) [inline src/styles/button.scss]: Line heights are always a fraction of the font-size and should never be an absolute pixel value.

**Review by dhruvkb — CHANGES_REQUESTED** [review]

**Comment by Dishebh** [comment]
@dhruvkb I have made some changes. Please let me know if they are fine

**Comment by Dhruvi16** [comment]
Thanks for contributing @Dishebh. We hope to see your other contributions. I would have to close this due to inactivity.

This issue is handled by - https://github.com/creativecommons/vocabulary/pull/565

**Pull request closed without merging**